contracts/drafts/Strings.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Strings
+ * @dev String operations.
+ */
+library Strings {
+    /**
+     * Concatenates two strings.
+     * string(abi.encodePacked(a, b))
+     */
+
+    /**
+     * @dev Converts a uint256 to a string.
+     * via OraclizeAPI - MIT licence
+     * https://github.com/oraclize/ethereum-api/blob/b42146b063c7d6ee1358846c198246239e9360e8/oraclizeAPI_0.4.25.sol
+     */
+    function uint256ToString(uint256 value) internal pure returns (string memory) {
```

Comment:
I'm not a fan of this name to be honest. Considering usage, what do you think about `fromUint256`? Calls would look like `Strings.fromUint256(15)`.